### PR TITLE
every curl needs optional s, and admin port

### PR DESCRIPTION
**Pull request by brntbeer**
I know we warn about this in a tip in the top, but it's so subtle and easily missed. Is there any reason not to include this elsewhere? cc @leefaus who was helping me with this!

cc @github/docs or @gjtorikian

### CODE REVIEW

**Comment by brntbeer** [comment]
also, just noticed the tip at the top of the page, but it's SO easy to miss. 

I guess alternatively you could change the`{tip}{/tip}` to be a `{warning}{/warning}` at the top.

**Comment by gjtorikian** [comment]
I'm definitely in favor of adding the `admin_port` to everything--thanks for that!

I'm not so sure about the `http(s)` bit, though. Is there any reason we can't just keep it as `http`, or `https`? I'm trying to think of the newbie admin who isn't entirely sure what the `(s)` means.

**Comment by brntbeer** [comment]
@gjtorikian not to mention, languages. maybe it'd look weird for certain people.

If the port is 8080, it should be known from set up that it's http, and 8443 for https. That's a decent compromise :wink:

**Comment by Janniesouta** [comment]
wesdrt

**Comment by Janniesouta** [comment]
https://patch-diff.githubusercontent.com/raw/github/developer.github.com/pull/859.patch

**Comment by brntbeer** [comment]
@gjtorikian i think we're set on this if you want to give it another look.

I also noticed some response `Location` sections missing `<em>` tags around `hostname`. and i added those admin_ports to them too

**Comment by leefaus** [comment]
:+1: Thanks to all for helping on this.  I think this will make it easier for enterprise customers to consume these endpoints.

File: content/v3/enterprise/management_console.md [hunk 1]
```diff
@@ -24,13 +24,13 @@ You need to pass your [Management Console password](https://help.github.com/ente
 Use the `api_key` parameter to send this token with each request. For example:
 
 <pre class="terminal">
-$ curl -L 'http://<em>hostname</em>/setup/api?api_key=<em>your-amazing-password</em>'
+$ curl -L 'http://<em>hostname</em>:<em>admin_port</em>:<em>admin_port</em>/setup/api?api_key=<em>your-amazing-password</em>'
```
Comment (gjtorikian) [inline content/v3/enterprise/management_console.md]: Duplicated port info.

**Comment by gjtorikian** [comment]
One tiny comment; when that's fixed feel free to merge. Thanks!

**Comment by brntbeer** [comment]
@gjtorikian great catch! updated

**Comment by brntbeer** [comment]
@gjtorikian just caught something, in other examples for enterprise, we put `(s)`. like on https://developer.github.com/v3/users/administration. Thoughts?

**Comment by gjtorikian** [comment]
Hmm. Since it's pointing to api.github.com, can we be bold and just switch 'em to `https`?

**Comment by leefaus** [comment]
@gjtorikian 

We have a number Enterprise customers that are not using HTTPS.

**Comment by kdaigle** [comment]
@gjtorikian can I get your :+1: on this if it is good with @leefaus' final comments? It looks like you have that last piece of outstanding feedback. :smile:

File: content/v3/enterprise/management_console.md [hunk 2]
```diff
@@ -24,13 +24,13 @@ You need to pass your [Management Console password](https://help.github.com/ente
 Use the `api_key` parameter to send this token with each request. For example:
 
 <pre class="terminal">
-$ curl -L 'http://<em>hostname</em>/setup/api?api_key=<em>your-amazing-password</em>'
+$ curl -L 'http://<em>hostname</em>:<em>admin_port</em>/setup/api?api_key=<em>your-amazing-password</em>'
```
Comment (mikemcquaid) [inline content/v3/enterprise/management_console.md]: Maybe `console_port` or mention `admin_port` specifically above?

**Comment by gjtorikian** [comment]
> @gjtorikian can I get your :+1: on this if it is good with @leefaus' final comments? It looks like you have that last piece of outstanding feedback. :smile:

I'm definitely against the existing `http(s)`, but let's also definitely punt on that for this PR.

I firmly believe The Right Thing to Do&trade; is:

1. Change `http(s)` to `https`.
2. [Change the Enterprise script](https://github.com/github/developer.github.com/blob/cb11b9bf5878071ff3c96bc12b9a0776c521489a/tasks/enterprise.rake) to replace `https` with `http`.

  We're already doing hella text substitutions whenever we generate the Enterprise docs, so we can add this one on too. I can take point for this work.

**Comment by kdaigle** [comment]
Thanks @gjtorikian! I merged this PR so you can tackle that work separately.

**Pull request merged**